app/views/shared/_cancel.html.slim

```diff
@@ -1,4 +1,6 @@
 - cancel = sign_up_or_idv_no_js_link || link
+- decorated_session = decorated_session
+- sp_name = decorated_session ? decorated_session.sp_name : ''
```

Comment:
This is not needed either because we have special decorator classes that will display the correct sp_name. You can safely call `decorated_session.sp_name` in a view and the correct value will be displayed.